src/libhoney.js

```diff
@@ -269,14 +272,24 @@ export default class Libhoney extends EventEmitter {
   }
 
   /**
-   * isClassic takes a writeKey and returns true if it is a "classic" writeKey,
-   * namely, that its length is exactly 32 characters.
+   * isClassic takes an API key and returns true if it is a "classic" Configuration API Key or Ingest API Key.
    * @returns {boolean} whether the key is classic
    *
-   * @private
+   * @example
+   *   if(libhoney.isClassic(apiKey)) {
+   *     // special case for classic environments
+   *   }
    */
-  isClassic(key) {
-    return key.length === 32;
+  static isClassic(key) {
+    if (key === null || key === undefined || key.length === 0) {
```

Comment:
other libhoneys, such as [libhoney-java](https://github.com/honeycombio/libhoney-java/blob/main/libhoney/src/main/java/io/honeycomb/libhoney/Options.java#L58) or [libhoney-go](https://github.com/honeycombio/libhoney-go/commit/60fd25a89a9779dfbfb031d19daf1fa0092bc9bf#diff-189ea9e27129a739de320a0975576142832f41aa7496b75232677043deec06e3L168), have a check that says if there is no key, it counts as classic. I've updated here to match, but given that this check wasn't here before I'm not sure if we want to keep it.